examples/textual_inversion/textual_inversion_mlflow.py

```diff
@@ -619,12 +608,12 @@ def main():
         diffusers.utils.logging.set_verbosity_error()
 
     # If passed along, set the training seed now.
-    if args.seed is not None:
+    if args.seed:
```

Comment:
You shouldn't do this. If `args.seed == 0` then it won't set seed, but `0` is still a valid seed